Linux_scripts/RHUI_repo_validation_scripts/repo_check.py

```diff
@@ -67,6 +85,8 @@ def yum_clean_up():
         stdout, stderr = yum_clean.communicate()
         if "Another app is currently holding the yum lock" in stderr.decode() or "Waiting for process with" in stderr.decode():
            msg_multiple_yum_process()
+     except:
+         logging.error("Issue happened while executing yum clean all command or while gathering yum processes list. Fix the issue and rerun the script")
```

Comment:
Following Error is encountered while executing...

Here we are talking about the yum errors and suggestions, right?

Reply:
If there is issue is getting yum process list due to performance issue or ps command got hung then this message will be printed.